documentation/modules/ROOT/pages/configuration/applying-transformations-selectively.adoc

```diff
@@ -0,0 +1,143 @@
+// Category: debezium-using
+// Type: assembly
+// ModuleId: applying-transformations-selectively-with-smt-predicates
+// Title: Applying transformations selectively with SMT predicates
+= Applying transformations selectively
+
+You can define a predicate for a single message transformation to specify a condition that event records must match before Kafka Connect applies the transformation.
+
+// Type: concept
+// ModuleID: about-smt-predicates
+// Title: About SMT predicates
+== SMT predicates
+
+{prodname} provides several single message transformations (SMTs) that you can use to modify event records before Kafka Connect saves the records to Kafka topics.
+By default, when you configure one of these SMTs for a {prodname} connector, Kafka Connect applies that transformation to every record that the connector emits.
+
+However, there can be instances in which you want to apply a transformation selectively, so that it modifies only that subset of change event messages that share a common characteristic.
+For example, you might want to run the transformation only on event messages from a specific table or that include a specific header key.
+In environments that run Kafka 2.6 or greater, you can append a predicate statement to a transformation to instruct Kafka Connect to apply the SMT only to event records that match a specified condition.
+When a predicate is configured, Kafka Connect evaluates each incoming message that it processes against the condition specified in the predicate.
+If the predicate condition is true for the event message, then the SMT is applied.
+Otherwise, the message is passed without any modifications.
+
+After you define a predicate, you can reuse it and apply it to other transforms.
+Predicates also include a `negate` option that you can use to invert a predicate so that the predicate condition is applied only to records that do not match the condition that is defined in the predicate statement.
```

Comment:
```suggestion
Predicates also include a `negate` option that you can use to invert a predicate so that the predicate condition is applied only to records that do _not_ match the condition that is defined in the predicate statement.
```

Perhaps highlight the _not_?